### PR TITLE
Lovely weather here

### PR DESCRIPTION
**Pull request by Salman93k**

### CODE REVIEW

**Comment by DTloss** [comment]
yeah, today is sunny day!

**Comment by Tarantula3000** [comment]
Congratulations on your first pull request, brilliant choice!

**Comment by b2017** [comment]
Congrats with your first GitHub contribution!!!

File: _pins/Salman93k.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Salman93k
+lattitude: 33.679913
```
Comment (impappl) [inline _pins/Salman93k.json]: One t!

**Comment by hollenberry** [comment]
@Salman93k , @impappl is correct! That's all you need to fix!

If you need any help, please check out the following video: https://vimeo.com/201011288, or reach out again and let us know if you have questions!

**Comment by crichID** [comment]
@Salman93k I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6694/commits/3830a6465288b8f03eabad4795e2ceabbf6b7cbd. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 26th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**